### PR TITLE
Reactivate normal Notifications for GW2024

### PR DESCRIPTION
**Pull request by shtukas**
Marketing have installed a new canvas for GW2024 ( https://github.com/guardian/membership-workflow/pull/509  ). We can now reactivate the normal flow for GW2024 (and stop using the direct Latcham link prototype Pascal had build).

### CODE REVIEW

**Review by tjmw — APPROVED** [review]
Looks good to me (though I don't have much context in this codebase!)

I think this removes an exception for the `GW2024` input, and after this gets merged it'll be treated like any other input?

**Comment by shtukas** [comment]
@tjmw 
Yep. I had essentially put the Notification step of GW2024 on hold, while we were working on the new Canvas :)

**Pull request merged**
